sotodlib/site_pipeline/preprocess_obs.py

```diff
@@ -184,7 +203,7 @@ def main(
         if tot_query=="":
             tot_query="1"
     
-    obs_list = context.obsdb.query(tot_query)
+    obs_list = context.obsdb.query(tot_query, tags=tags)
```

Comment:
Have you tested passing a list of tags i.e. `jupiter, saturn, moon`? And perhaps its more intuitive to just pass the SSO name to the argparse instead of `<sso_name>=1` and deal with the `=1` bit when you pass the tags to `obsdb.query`? Also I think this is case-sensitive can you add a `<str>.lower()` (I think it has to be all lower case) so this doesn't break if someone passing `Jupiter` instead of `jupiter`?